exporters/Makefile

```diff
@@ -0,0 +1,54 @@
+PWD                   = $(shell pwd)
+NODE_EXPORTER_VERSION = 1.11.1
+KUBE_STATE_METRICS    = 2.8.11
+RELEASE_CHANNEL = "stable"
+
+
+
+.PHONY: build
+build: kube-state-metrics prometheus-node-exporter
+
+.PHONY: template
+template:
+	@mkdir -p vanillas
+	helm template ${CHART} ${RELEASE_CHANNEL}/${CHART} --version=${VERSION} > vanillas/${CHART}.yaml
+	helm template ${CHART} ${RELEASE_CHANNEL}/${CHART} --version=${VERSION} --output-dir .
+
+.PHONY: kube-state-metrics
+kube-state-metrics: docker-gen-kube-state-metrics
+
+.PHONY: gen-kube-state-metrics
+gen-kube-state-metrics: kube-state-metrics-template kube-state-metrics/kustomization.yaml
+
+.PHONY: kube-state-metrics-template
+kube-state-metrics-template:
+	make template CHART=kube-state-metrics VERSION=${KUBE_STATE_METRICS}
```

Comment:
These seem a little confusing to me, like I think the following is a bit clearer, also it introduces a comment/suggestion I've made below to ditch the vanillas folder and just put them in the same folder as the kustomizaton.yaml

Here is my suggestion...

```
PWD                              = $(shell pwd)
PROMETHEUS_NODE_EXPORTER_VERSION = 1.11.1
KUBE_STATE_METRICS_VERSION       = 2.8.11
RELEASE_CHANNEL                  = "stable"


.PHONY: build
build: kube-state-metrics prometheus-node-exporter

.PHONY: template
template:
	helm template ${CHART} ${RELEASE_CHANNEL}/${CHART} --version=${VERSION} > ${CHART}/vanilla-${VERSION}.yaml
	helm template ${CHART} ${RELEASE_CHANNEL}/${CHART} --version=${VERSION} --output-dir .

kube-state-metrics/vanilla-%.yaml:
	make template CHART=kube-state-metrics VERSION=${KUBE_STATE_METRICS_VERSION}

.PHONY: kube-state-metrics-docker
kube-state-metrics-docker: docker-kube-state-metrics

.PHONY: kube-state-metrics
kube-state-metrics: kube-state-metrics/vanilla-${KUBE_STATE_METRICS_VERSION}.yaml kube-state-metrics/kustomization.yaml

prometheus-node-exporter/vanilla-%.yaml:
	make template CHART=prometheus-node-exporter VERSION=${PROMETHEUS_NODE_EXPORTER_VERSION}

.PHONY: prometheus-node-exporter-docker
prometheus-node-exporter-docker: docker-prometheus-node-exporter

.PHONY: prometheus-node-exporter
prometheus-node-exporter: prometheus-node-exporter/vanilla-${PROMETHEUS_NODE_EXPORTER_VERSION}.yaml prometheus-node-exporter/kustomization.yaml

.SECONDEXPANSION:
%/kustomization.yaml: $$(wildcard $$*/templates/*.yaml)
	@echo $^
	./kustomization.yaml.sh $(shell cd $* && ls templates/*) > $@

.dockerimage: Dockerfile
	docker build -t manifests-builder .
	touch .dockerimage

.PHONY: docker-%
docker-%: .dockerimage
	docker \
      run \
      --rm \
      -it \
      -v ${PWD}/:/workdir \
      manifests-builder \
	  make $*
```